rules/windows/process_creation/proc_creation_win_renamed_vmnat.yml

```diff
@@ -1,11 +1,12 @@
-title: Renamed or Portable Vmnat.exe
+title: Renamed Vmnat.exe Execution
```

Comment:
You can remove `filter_portable` as if it is true, `filter_rename` will also be true.